inputfiles/idl/Web Animations.widl

```diff
@@ -93,22 +93,22 @@ interface KeyframeEffect : AnimationEffect {
 };
 
 dictionary BaseComputedKeyframe {
-     double?             offset = null;
-     double              computedOffset;
-     DOMString           easing = "linear";
-     CompositeOperation? composite = null;
+     double?                  offset = null;
+     double                   computedOffset;
+     DOMString                easing = "linear";
+     CompositeOperationOrAuto composite = "auto";
```

Comment:
Are these `.widl` files supposed to be hand-edited? Next time anyone runs the `fetcher` script, they'll get overridden, so I would imagine you want to use `overridingTypes.json` to achieve what you want.

Reply:
Hi Daniel, thanks for looking at this PR. Please see the comment of this commit (the first of the two). That is, this is literally the result of running the fetcher script so this will not get overridden next time someone runs it.